packages/frontend/src/scripts/transaction/sign.ts

```diff
@@ -0,0 +1,25 @@
+import { EthereumAddress } from '@explorer/types'
+
+import { ForcedTradeInitialOfferEntry } from './types'
+
+export async function signInitial(
+  offer: ForcedTradeInitialOfferEntry,
+  address: EthereumAddress
+) {
+  const provider = window.ethereum
+
+  if (!provider || !address) {
+    return
+  }
+
+  const stringOffer = JSON.stringify(offer, null, 2)
```

Comment:
Again, I'd prefer explicit here, because then the field order (and count, as TS doesn't prevent additional fields) is guaranteed.